dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImpl.java

```diff
@@ -160,7 +160,7 @@ public String saveContent(final Map<String, Object> contentletFormData,
 			// before DOTCMS-6383
 			//conAPI.unlock(cont, user, false);
 			if (null != contentlet) {
-				this.pushSaveEvent(contentlet, isNew);
+				contentletSystemEventUtil.pushSaveEvent(contentlet, isNew);

```

Comment:
this won't work, b/c Angular is gonna get the event immediate and them retrieve the info and won't be updated yet

Reply:
@jdotcms what was the original issue that this change was supposed to solve?

Reply:
I imagine a race condition angular getting the dirty version of the content @wezell

Reply:
could this be fixed by the new work being done by @jdotcms ?

Reply:
Right, I get all of that.  I was just wondering where this was an issue with our current system rather than a theoretical issue.